tools/submission/submission_checker.py

```diff
@@ -1309,6 +1509,7 @@ def __init__(
         self.seeds = self.base["seeds"]
         self.test05_seeds = self.base["test05_seeds"]
         self.accuracy_target = self.base["accuracy-target"]
+        self.accuracy_upper_limit = self.base["accuracy-upper-limit"]
```

Comment:
I think we should use self.base.get("accuracy-upper-limit") as it is not there for all the benchmarks.

Reply:
This is done here: https://github.com/pgmpablo157321/inference/blob/f7c755a0e156f51c1684f984889b309b8afaffc2/tools/submission/submission_checker.py#L1618

The script passes the dictionary to the config and calls get for each model